rafx-framework/src/visibility/entity_id.rs

```diff
@@ -0,0 +1,25 @@
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
+pub struct EntityId(u64);
```

Comment:
I think this is a case where, if we had a prefix on names, it would be helpful to end-users because they wouldn't have to deal with having two EntityId structs simultaneously in their code. I don't have a specific suggestion for that though and it's probably a question that's outside the scope of this PR.

Reply:
`OpaqueId`? `CallbackId`?

Reply:
I lean towards either UserId or just stick with EntityId. None of them feel vastly better than the others to me.

Reply:
Renamed to `UserId`.